src/components/views/elements/PersistedElement.tsx

```diff
@@ -184,7 +184,7 @@ export default class PersistedElement extends React.Component<IProps> {
             width: parentRect.width + 'px',
             height: parentRect.height + 'px',
         });
-    }, 100, { trailing: true, leading: true });
+    }, 16, { trailing: true, leading: true });
```

Comment:
Is this okay?
Can I just increase the throttled calls? (This is done because otherwise the jitsi dom element lacks behind the container (A LOT) )